HISTORY.rst

```diff
@@ -4,6 +4,14 @@
 History
 =======
 
+v0.5.0 (unreleased)
+-------------------
+Contributors to this version: Juliette Lavoie (:user: `juliettelavoie`)
+
+New features
+^^^^^^^^^^^^
+* Add a template for NEX-GDDP-CMIP6. (:pull:`120`).
```

Comment:
I don't have the special sphinx documentation operators in miranda. Maybe it's time to add them?

Reply:
ah I don't think it's necessary. I just thought it was the same as the other packages.

Reply:
Honestly, I should look into making a sphinx plugin for it at this point; So many projects use/could benefit from this.